homeassistant/components/xiaomi_miio/vacuum.py

```diff
@@ -293,6 +301,27 @@ def fan_speed_list(self):
         """Get the list of available fan speed steps of the vacuum cleaner."""
         return list(self._fan_speeds)
 
+    @property
+    def timers(self):
+        """Get the list of added timers of the vacuum cleaner."""
+        if self.timezone not in all_timezones:
+            return []
+
+        timers = []
+        local_tz = timezone(self.timezone)
+
+        for timer in self._timers:
+            cron = croniter(timer.cron, start_time=local_tz.localize(datetime.now()))
```

Comment:
I think it would make sense to move this parsing to the backend library, do you agree? I think the data structure is fine, although you may want to expose the ID also, just in case if the API gets improved at some point to delete timers / toggle their state.

Reply:
Yes I agree. Sounds good, I will create a pull request in python-miio library.

Reply:
@MarBra I just release python-miio 0.5.1, feel free to continue on this PR! :)